src/index.js

```diff
@@ -0,0 +1,5 @@
+import createMainPage from './main-page';
+
+createMainPage();
+
+console.log('I am created inside index, inside src');
```

Comment:
- [ ] [required]  No console.log committed to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png) from Pathwright)